test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -174,7 +176,7 @@ def _test_common(
             with torch.no_grad(), maybe_autocast:
                 clone_inputs = self._clone_inputs(inputs)
                 expected = mod(*inputs)
-                actual = torch.compile(mod)(*clone_inputs)
+                actual = torch.compile(mod, **compile_options)(*clone_inputs)
```

Comment:
@Xia-Weiwen, how about creating a new UT subclass for this PR? 
This PR currently seems to require manual intervention because line 177 on the left does not seem to be present in Meta-internal code -
<img width="1502" alt="image" src="https://github.com/user-attachments/assets/32cb1ecb-b6e5-423d-8c71-6ea1a55409de">

So, if you'd create a new subclass of `TestPatternMatcherBase` and would overload `_test_code_common`, then this PR's code wouldn't require manual-changes in Meta-internal code, and the `Meta Internal-Only Changes Check` CI check will pass.

We might be able to land this PR before v2.6 code freeze with such a change.

Reply:
Sorry, just noticed that `Meta Internal-Only Changes Check` had initially passed with this PR.